src/utilities.h

```diff
@@ -656,11 +656,11 @@ bool traverseComponentEntityTreeLinkingUnits(const ComponentEntityPtr &component
  *  Utility function used when linking units names to their corresponding @c Units items.
```

Comment:
@ref Units, not @c.

Reply:
I am inclined to leave it for another issue? I mean, I just searched for `@c Units` in our codebase and there are many instances of it (and other similar ones).

Reply:
Happy to change this particular one though, if you really insist. :)

Reply:
Agreed that we should try and fix this in bulk. Maybe a session during a meeting? perhaps.  I have been fixing them up if I notice them for any functions or methods that I change.  I won't insist but I think it would be better to have it done than not.

Reply:
Done in commit b904f477.